docs/cpp/cpp-ide.md

```diff
@@ -49,61 +49,81 @@ ColumnLimit: 0
 
 To use a different version of clang-format than the one that ships with the extension, change the `C_Cpp.clang_format_path` [setting](/docs/getstarted/settings.md) to the path where the clang-format binary is installed.
 
-For example, on the Windows platform:
+For example, on the Windows platform, use:
 
 ```json
   "C_Cpp.clang_format_path": "C:\\Program Files (x86)\\LLVM\\bin\\clang-format.exe"
 ```
 
 ### Enhanced semantic colorization
 
-When IntelliSense is enabled, the Visual Studio Code C/C++ extension supports semantic colorization. See [Enhanced colorization](/docs/cpp/colorization-cpp.md) for more details about setting colors for classes, functions, variables and so on.
+When IntelliSense is enabled, the Visual Studio Code C/C++ extension supports semantic colorization. For more information about setting colors for classes, functions, variables, etc., see [Enhanced colorization](/docs/cpp/colorization-cpp.md). For more information on configuring IntelliSense, see [IntelliSense configuration](/docs/cpp/configure-intellisense.md).
 
 ### Quick Info
 
 You can hover over a symbol to see an inline view of its definition:
 
 ![Quick info](images/mingw/quickinfo.png)
```

Comment:
Perhaps we can add information about documentation comments under this section since they show up in the tooltip?

For example, Doxygen Generation and formatting in the tooltip. An example of the usage of doxygen tags can be found here: 

```
/// @brief Calculates area of triangle
/// @tparam T is template param
/// @param base is horizontal length
/// @param height is vertical length
/// @return Area of triangle
/// @exception This is an exception comment
/// @deprecated This is deprecated comment
/// @note this is note
/// @attention this is attention
/// @pre this is pre comment
template<typename T>
T TriangleArea(T base, T height)
{
    double result;
    result = base * height * 0.5;
    return (T)result;
}
```

You could also mention the following settings: 

![image](https://github.com/microsoft/vscode-docs/assets/111317156/efe58deb-56b0-45fc-8564-4f2d435c6fc8)